packages/front-end/pages/metric/[mid].tsx

```diff
@@ -861,22 +861,23 @@ const MetricPage: FC = () => {
                   <li className="mb-2">
                     <span className="text-gray">Minimum sample size:</span>{" "}
                     <span className="font-weight-bold">
-                      {metric?.minSampleSize ?? defaultMinSampleSize}
+                      {metric?.minSampleSize ||
+                        metricDefaults.minimumSampleSize}
```

Comment:
This won't work if someone sets `metric.minSampleSize` to 0.  I'm not 100% sure if that's even allowed in the UI, so maybe not a real problem.

Reply:
We do this same type of fallback logic all over the place.  I wonder if it would make more sense to have something like this instead:

```ts
metricDefaults.getMinSampleSize(metric)
```

Reply:
I've addressed this with the updates to the hook. It now exposes some helper methods to ensure 0 is not overlooked.